Classifiers/classifier-Prisma_Cloud.json

```diff
@@ -1505,6 +1507,494 @@
                                 }
                         }
                 },
+		"GCP Compute Engine Misconfiguration": {
+                        "dontMapEventToLabels": false,
+                        "internalMapping": {
+                                "Account Name": {
+                                        "complex": {
+                                                "accessor": "account",
```

Comment:
Please replace account field in all relevant places

Reply:
@mayagoldb I'm not sure what you're asking?  I believe `"Account Name:"` is correct, right?